src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala

```diff
@@ -13,38 +20,90 @@ object ToolFormApp extends App {
     * @param args Command line arguments.
     * @return Either[String, Config] - The configuration or an error.
     */
-  def execute(args: Array[String]): Either[String, Config] = {
-    val parser = new scopt.OptionParser[CommandConfig]("toolform") {
+  def execute(args: Array[String]): Either[ToolFormError, String] =
+    for {
+      toolConfiguration <- parseCommandLineArgs(args)
+      project           <- readProject(toolConfiguration.in)
+      summary           <- projectSummary(project)
```

Comment:
@dspasojevic Maybe a problem for the future, but with the for-comp here, there doesn't seem to be a great way to dispatch to different commands.  Currently it goes in one direction and produces the inspect summary, but if I want a support command#2, it seems like that will really break up the party.  Thoughts?

Reply:
Yep. Good observation. This may change to something like:
```
for {
   ...
   command <- configuredCommand(toolConfiguration, project)
   output      <- execute(command)
} yield output
```